<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>file-uploader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="file-uploader.qrc">
    <normaloff>:/img/app.png</normaloff>:/img/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="downloadTab">
        <attribute name="title">
         <string>Загрузка файлов</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableView" name="urlsView">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>24</number>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="doWorkBtn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Загрузить</string>
             </property>
             <property name="icon">
              <iconset resource="file-uploader.qrc">
               <normaloff>:/img/download.png</normaloff>:/img/download.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="loggerEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1325</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openUrlsAction"/>
    <addaction name="exportUrlsAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="paramsAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openUrlsAction"/>
   <addaction name="exportUrlsAction"/>
  </widget>
  <action name="openUrlsAction">
   <property name="icon">
    <iconset resource="file-uploader.qrc">
     <normaloff>:/img/import-file.png</normaloff>:/img/import-file.png</iconset>
   </property>
   <property name="text">
    <string>Открыть URLs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="exportUrlsAction">
   <property name="icon">
    <iconset resource="file-uploader.qrc">
     <normaloff>:/img/export-file.png</normaloff>:/img/export-file.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт URLs</string>
   </property>
   <property name="toolTip">
    <string>Экспорт ссылок на изображения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="paramsAction">
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="file-uploader.qrc"/>
 </resources>
 <connections/>
</ui>
